xcube/core/store/descriptor.py

```diff
@@ -129,8 +155,10 @@ def __init__(self,
                  spatial_res: float = None,
                  time_range: Tuple[Optional[str], Optional[str]] = None,
                  time_period: str = None,
+                 coords: Sequence[str] = None,
                  dims: Mapping[str, int] = None,
-                 data_vars: Sequence['VariableDescriptor'] = None,
+                 data_vars: Union[Sequence['VariableDescriptor'],
+                                  Mapping[str, 'VariableDescriptor']] = None,
```

Comment:
Get rid of `Sequence`.

```suggestion
                 data_vars: Mapping[str, 'VariableDescriptor'] = None,
```